MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1440,6 +1441,9 @@
 			_metalFeatures.maxPerStageDynamicMTLBufferCount = _metalFeatures.maxPerStageBufferCount;
 			_metalFeatures.postDepthCoverage = true;
 			_metalFeatures.renderLinearTextures = true;
+			// Apple devices don't like these.
+			_metalFeatures.memoryBarriers = false;
+			_metalFeatures.textureBarriers = false;
```

Comment:
It suppose it might mean more setting lines...but can you flip this around so that all settings are positively set, please?

The pattern on these settings is to start with everything disabled and then selectively enable them when appropriate. None of the other settings are enabled and then disabled again like this. It feels like this could become spaghetti-like and fragile over time.

Reply:
Done.